src/services/companyOffersService.js

```diff
@@ -1,97 +1,12 @@
-import Offer from "../components/HomePage/SearchResultsArea/Offer/Offer";
+import config from "../config";
+const { API_HOSTNAME } = config;
 
 
-// import config from "../config";
-// const { API_HOSTNAME } = config;
+const parseFiltersToURL = (filters) => Object.keys(filters).map((key) => `${key}=${filters[key]}`).join("&");
```

Comment:
A similar function is defined [here](https://github.com/NIAEFEUP/nijobs-fe/blob/develop/src/services/offerService.js#L7). Maybe you can reuse it?

Reply:
I believe it can be reused as well. although for now, the service is not using filters inside the request, it is there for future implementation. Right now, there is the possibility of filtering, but only after fetching the data

Reply:
But I deleted the duplicated method already to avoid confusion in the future